src/supervised/scaledloss.jl

```diff
@@ -1,4 +1,4 @@
-for KIND in (:MarginLoss, :DistanceLoss)
+for KIND in (:MarginLoss, :DistanceLoss, :SupervisedLoss)
```

Comment:
This was actually a mistake -- I tried this out and decided against it. I'll revert back in the next commit?